saleor/payment/gateways/stripe/__init__.py

```diff
@@ -27,6 +27,10 @@ def get_client_token(**_):
     return
 
 
+def get_public_key(config: GatewayConfig):
```

Comment:
I need public key for Stripe template to render payment with elements.
`var stripe = Stripe('{{public_key}}');`

Reply:
Yes I know :) public_key is accessible in payment_config. See in plugin.py:
```python
@require_active_plugin
def get_payment_config(self, previous_value):
    config = self._get_gateway_config()
        return [
            {"field": "api_key", "value": config.connection_params["public_key"]},
            {"field": "store_customer_card", "value": config.store_customer},
        ]
```
You can ask payment plugin for it directly, we use it in API too, so it would be consistent.

Reply:
In this case, public_key, is the same of Braintree's `client_token` (Braintree template)